specs/language/introduction.tex

```diff
@@ -0,0 +1,165 @@
+\Ch{Introduction}{Intro}
+
+\p The \acrfull{hlsl} is the GPU programming language provided in conjunction
+with the \gls{dx} runtime. Over many years its use has expanded to cover every
+major rendering API across all major development platforms. Despite its
+popularity and long history \acrshort{hlsl} has never had a formal language
+specification. This document seeks to change that.
+
+\p \acrshort{hlsl} draws heavy inspiration originally from \gls{isoC} and later
+from \gls{isoCPP} with additions specific to graphics and parallel computation
+programming. The language is also influenced to a lesser degree by other popular
+graphics and parallel programming languages.
+
+\p \acrshort{hlsl} has two reference implementations which this specification
+draws heavily from. The original reference implementation \acrfull{fxc} has been
+in use since \gls{dx} 9. The more recent reference implementation \acrfull{dxc}
+has been the primary shader compiler since \gls{dx} 12.
+
+\p In writing this specification bias is leaned toward the language
+behavior of \acrshort{dxc} rather than the behavior of \acrshort{fxc}, although
+that can vary by context.
+
+\p In very rare instances this spec will be aspirational, and may diverge from
+both reference implementation behaviors. This will only be done in instances
+where there is an intent to alter implementation behavior in the future. Since
+this document and the implementations are living sources, one or the other may
+be ahead in different regards at any point in time.
+
+\Sec{Scope}{Intro.Scope}
+
+\p This document specifies the requirements for implementations of
+\acrshort{hlsl}. The \acrshort{hlsl} specification is based on and highly
+influenced by the specifications for the \acrfull{c} and the \acrfull{cpp}.
+
+\p This document covers both describing the language grammar and semantics for
+\acrshort{hlsl}, and (in later sections) the standard library of data types used
+in shader programming.
+
+\Sec{Normative References}{Intro.Refs}
+
+\p The following referenced documents provide significant influence on this
+document and should be used in conjunction with interpreting this standard.
+
+\begin{itemize}
+  \item \gls{isoC}, \textit{Programming languages - C}
+  \item \gls{isoCPP}, \textit{Programming languages - C++}
+  \item \gls{dx} Specifications, \textit{https://microsoft.github.io/DirectX-Specs/}
+\end{itemize}
+
+\Sec{Terms and definitions}{Intro.Terms}
+
+\p This document aims to use terms consistent with their definitions in
+\gls{isoC} and \gls{isoCPP}. In cases where the definitions are unclear, or
+where this document diverges this section, the remaining sections in this
+chapter, and the attached \ref{glossaries}.
+
+\Sec{Runtime Targeting}{Intro.Runtime}
+
+\p \acrshort{hlsl} evolved out of \gls{dx} and gained popularity because it
```

Comment:
The phrasing here makes it sound like HLSL replaced DirectX. I think something with "emerged" might communicate the intended message. "HLSL emerged from the evolution of DX to grant greater and greater control over the specifics of the GPU geometry and color processing. . . ". A bit wordy.

Reply:
Change incoming.